lib/data/copy/concerns/mirrorable.yml

```diff
@@ -1,7 +1,7 @@
-mirror_msg:
+mirror_plain:
```

Comment:
Use `team.bot_mention` instead of this `slack_mention` variable throughout this file.